scripts/load_meeting_prot.py

```diff
@@ -1,4 +1,7 @@
-"""Script to load info from trello and merge with info from latest meeting protocol."""
+"""This script loada info from trello and merge with info from latest meeting 
+protocol. It then loads a new meeting protocol with the merged info devided into
+comming and ongoing deliveries. The info consists of project name,project info, 
+flow cell ids and comments."""
 
 import bcbio.google
```

Comment:
Good point @guillermo-carrasco , is also what @mayabrandi is concerned about. However there is a clear value in the spread sheet we sued to check the ongoing deliveries, it should not be difficult to move it to genstats but in this moment we have no time to dedicate to it. 
For now this script is helping out a lot